de/portal.json

```diff
@@ -74,7 +74,7 @@
             "Please verify email address": "Bitte überprüfen Sie Ihre E-Mail Adresse, um auf alle Funktionen von Sinric zugreifen zu können",
             "devices": "Geräte",
             "Ding Dong": "Ding Dong!",
-            "Power is": "Macht ist",  // What is meant with this? It makes no sense in German.
+            "Power is": "Macht ist",
```

Comment:
Hi Jxnathan! 
"Power is on"  /  "Power is off". Im deutschen würden wir aber eher wohl sagen "Das Gerät ist eingeschaltet" / "Das Gerät ist ausgeschaltet"

Reply:
Ja, daher die Frage weil das so in der Übersetzung stand.